test/test-utils/wrappers.tsx

```diff
@@ -57,3 +58,11 @@ export function wrapInSdkContext<T>(
         }
     };
 }
+
+export function renderWithClientContextOptions(client: MatrixClient): RenderOptions {
```

Comment:
doc-comment would be nice. What is the intended usage of this, and what does it do?

Reply:
:thinking: should it be called `withClientContextRenderOptions`? The current name implies it does the rendering itself, rather than just preparing an options object.